plugins/linux-v4l2/v4l2-udev.c

```diff
@@ -157,6 +164,10 @@ void v4l2_init_udev(void)
 	if (udev_refs == 0) {
 		if (os_event_init(&udev_event, OS_EVENT_TYPE_MANUAL) != 0)
 			goto fail;
+		if ((udev_event_fd = eventfd(0, EFD_CLOEXEC)) < 0) {
+			close(udev_event_fd);
```

Comment:
On error, `udev_event_fd` will be -1 and this `close()` call will error out with `EBADF` I think. Seems like you don't need to close this here.

Reply:
Left the close in the wrong branches, that should be fixed up.